jobs/tendrl-jobs-build-triggers.yaml

```diff
@@ -0,0 +1,77 @@
+- job-template:
+    name: "tendrl-build-{package_name}"
+    display-name: 'Tendrl-{package_name}'
+    description: 'Do not edit this job through the web!'
+    project-type: freestyle
+    defaults: global
+    disabled: false
+    concurrent: false
+    quiet-period: 0
+    block-downstream: false
+    block-upstream: false
+    node: 'master'
+    triggers:
+        - timed: 'H H(0-7) * * *'
+        - pollscm:
+            cron: 'H/10 * * * *'
+            ignore-post-commit-hooks: false
```

Comment:
Same as above, this was initially just copy&paste (rewrite) of the original jobs to job builder yaml, to have the configuration in git (with the same functionality as it is configured in the existing jobs). But I agree that it doesn't make much sense.